docs/antispam.rst

```diff
@@ -43,6 +43,8 @@ Rspamd rejects non-compliant email messages and email messages that contain viru
       
    66% (10/15) is less than 80%, so the email is classified as ham. This email message will go to the inbox folder. If the user wants email messages with a score of 10 (66%) to be classified as spam, then the user defined spam filter tolerance can be lowered to 65% in the administration web interface.
    
+   The default spam filter tolerance used for new users can be configured using the environment variable ``DEFAULT_SPAM_THRESHOLD``. See also: :ref:`_advanced_cfg`.
```

Comment:
The build fails with the following error for the documentation:
`/docs/antispam.rst:46:undefined label: _advanced_cfg`

When referring to a bookmark you should not include the leading _ of the bookmark. 
Try changing ```:ref:`_advanced_cfg`.``` to ```:ref:`advanced_cfg` ```.

In the future I also recommend to try building and testing the images manually before submitting a PR or pushing commits to a PR. How to is described here:
https://mailu.io/1.9/contributors/environment.html#building-images
This way you can see beforehand that the build fails..